### PR TITLE
fix(weekly): update path resolution

### PR DESCRIPTION
**Pull request by Steveantor**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "build": "tsc",
     "build:ci": "ncc build src/index.ts -o dist && mv ./dist/index.js ./bin",
     "start": "probot run ./lib/main.js",
-    "start:watch": "nodemon --exec 'yarn start'",
+    "start:watch": "copy-folder-util src/assets/images lib/assets/images && nodemon --exec 'yarn start'",
```
Comment (pavlovcik) [inline package.json:17]: This seems really unnecessary to add this dependency. What's the issue with `cp -r` or `rsync`

**Review by pavlovcik — CHANGES_REQUESTED** [review]
I generally disagree with adding dependencies unless they are really needed. In this case, I don't see why we can't use built in binaries like `cp`

File: package.json (hunk 1 above)
Comment (Steveantor) [reply package.json:17]: I'm thinking about cross platform support, whichever command or dependency works
@pavlovcik

Comment (pavlovcik) [reply package.json:17]: I thought that `cp` works for windows lol

**Comment by pavlovcik** [comment]
Let me one up you with an idea, why not use a symlink instead of cp for this?

**Comment by pavlovcik** [comment]
Mixed feelings about merging that in but its such a minor thing I figured no need to block you on it.

**Pull request merged**